src/custom/pages/LimitOrder/pureComponents/CurrencyInputPanel/styled.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components/macro'
+import { loadingOpacityMixin } from '@src/components/Loader/styled'
+import Input from '@src/components/NumericalInput'
```

Comment:
Any reason it must be from `@src/`?

Unless you really mean to import from Uni's, avoid it

Reply:
Fixed: https://github.com/cowprotocol/cowswap/pull/1049/commits/6234dc332de799d2a559ef94e6cce1fa72ce76ee
p.s. Idea generates this code